doc/MANUAL.txt

```diff
@@ -226,17 +268,36 @@ class (the last . splits the class and test method)::
     group_regex=([^\.]+\.)+
 
 
-Remote or isolated test environments
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Profiles and remote or isolated test environments
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-A common problem with parallel test running is test runners that use global
-resources such as well known ports, well known database names or predictable
-directories on disk. 
+A common testing problem is testing the same code against a matrix of
+test scenarios. For instance with a C program you may wish to test against
```

Comment:
This term clashes with "testscenarios". I guess they are kind of the same thing, so I'm not sure it matters. I guess the message is "use testscenarios for some scenarios, but testr profiles for other scenarios"?